src/app/rpmostree-builtin-deploy.cxx

```diff
@@ -119,6 +122,10 @@ rpmostree_builtin_deploy (int            argc,
     }
   else
     {
+      if (!opt_bypass_driver)
+        if (!error_if_driver_registered (bus_type, sysroot_proxy, cancellable, error))
```

Comment:
It'd be safer to use braces here for the outer `if-statement`, but in this case you can also just combine the two, e.g.

```suggestion
      if (!opt_bypass_driver && !error_if_driver_registered (bus_type, sysroot_proxy, cancellable, error))
```

Reply:
Would combining the two make it a bit less clear what is going on?
I've added braces for the outer `if-statement` for now, but let me know if combining makes more sense.

Reply:
Nope, totally fine as is too! I'm just used to picking up on those patterns.